Documentation/EspeakFramework.md

```diff
@@ -0,0 +1,17 @@
+# eSpeak-NG Framework Packaging
```

Comment:
can you double check the license - some may require us to include their copy of the license in our code base. In which case you could 

<img width="1084" height="969" alt="Image" src="https://github.com/user-attachments/assets/e24919db-5e5a-4eac-8e9b-7adc17d1d300" />